web/src/features/Auth/components/SecureAdminConsole.tsx

```diff
@@ -1,12 +1,39 @@
 import * as React from "react";
 import Helmet from "react-helmet";
-import { Utilities, dynamicallyResizeText } from "../utilities/utilities";
-import Loader from "./shared/Loader";
-import ErrorModal from "./modals/ErrorModal";
+import { Utilities, dynamicallyResizeText } from "@src/utilities/utilities";
+import Loader from "@src/components/shared/Loader";
+import ErrorModal from "@src/components/modals/ErrorModal";
 import "../scss/components/Login.scss";
```

Comment:
some fixes is needed for Login.scss import in SecureAdminConsole, check build-web logs